docs/src/main/paradox/docs/getting-started/running-nexus/configuration/index.md

```diff
@@ -0,0 +1,91 @@
+
+## Plugins configuration
+
+Since 1.5.0, Nexus Delta supports plugins. Jar files present inside the local directory defined by the `DELTA_PLUGINS` environment variable are loaded as plugins into the Delta service. 
+
+Each plugin configuration is rooted under `plugins.{plugin_name}`. All plugins have a `plugins.{plugin_name}.priority` configuration flag used to determine the order in which the routes are handled in case of collisions. 
+
+### Elasticsearch views plugin configuration
+
+The elasticsearch plugin configuration can be found @link:[here](https://github.com/BlueBrain/nexus/blob/master/delta/plugins/elasticsearch/src/main/resources/elasticsearch.conf){ open=new }. 
+
+The most important flag is `plugins.elasticsearch.base` which defines the endpoint where the Elasticsearch service is running.
+
+### Blazegraph views plugin configuration
+
+The blazegraph plugin configuration can be found @link:[here](https://github.com/BlueBrain/nexus/blob/master/delta/plugins/blazegraph/src/main/resources/blazegraph.conf){ open=new }. 
+
+The most important flag is `plugins.blazegraph.base` which defines the endpoint where the Blazegraph service is running.
+
+### Composite views plugin configuration
+
+The composite views plugin configuration can be found @link:[here](https://github.com/BlueBrain/nexus/blob/master/delta/plugins/composite-views/src/main/resources/composite-views.conf){ open=new }. 
+
+There are several configuration flags related to tweaking the range of values allowed for sources, projections and rebuild interval.
+
+### Storage plugin configuration
+
+The storage plugin configuration can be found @link:[here](https://github.com/BlueBrain/nexus/blob/master/delta/plugins/storage/src/main/resources/storage.conf){ open=new }. 
+
+Nexus Delta supports 3 types of storages: 'disk', 'amazon' (s3 compatible) and 'remote'.
+
+- For disk storages the most relevant configuration flag is `plugins.storage.storages.disk.default-volume`, which defines the default location in the Nexus Delta filesystem where the files using that storage are going to be saved.
+- For S3 compatible storages the most relevant configuration flags are the ones related to the S3 settings: `plugins.storage.storages.amazon.default-endpoint`, `plugins.storage.storages.amazon.default-access-key` and `plugins.storage.storages.amazon.default-secret-key`.
+- For remote disk storages the most relevant configuration flags are `plugins.storage.storages.remote-disk.default-endpoint` (the endpoint where the remote storage service is running) and `plugins.storage.storages.remote-disk.default-credentials` (the Bearer token to authenticate to the remote storage service).
+
+
+### Archive plugin configuration
+
+The archive plugin configuration can be found @link:[here](https://github.com/BlueBrain/nexus/blob/master/delta/plugins/archive/src/main/resources/archive.conf){ open=new }.
```

Comment:
It is missing a word about monitoring/kamon.
How to enable/disable it, a link to the default conf and a link to the kamon website for example

Reply:
I have no idea about kamon. Someone else can fill in that part afterwards.

Reply:
The env variables, a link to kamon.conf and links to the kamon, jaeger, promotheus websites.

If you don't want it to be part of this PR, open a new issue about monitoring configuration

Reply:
Done https://github.com/BlueBrain/nexus/issues/2411